test/index.html

```diff
@@ -16,22 +16,16 @@
   <script src="/node_modules/video.js/dist/video.js"></script>
   <script src="/node_modules/videojs-contrib-media-sources/src/videojs-media-sources.js"></script>
 
-  <script src="/src/videojs-hls.js"></script>
+  <script src="/src/videojs-contrib-hls.js"></script>
   <script src="/src/xhr.js"></script>
-  <script src="/src/stream.js"></script>
-  <script src="/src/m3u8/m3u8-parser.js"></script>
+  <script src="/dist/videojs-contrib-hls.js"></script>
```

Comment:
yeah dist is the only thing that we will need to include in the future. Right now it only contains stub.js (which adds our objects to HLS directly so that the unit tests will still work), m3u8, and stream. As we go through the stages things will be removed from here and into that dist file.